docs/verifiable-document/dns-proof.md

```diff
@@ -12,52 +12,70 @@ In this guide, we will bind the document issuer's identity to a valid domain nam
 
-```text
-openatts net=ethereum netId=3 addr=&lt;DOCUMENT_STORE_ADDRESS&gt;
-```
+Within your domain registrar or DNS provider's web UI, insert a `TXT` record into the DNS in the following format:
 
-where `<DOCUMENT_STORE_ADDRESS>` is the document store smart contract address obtained [in the previous guide](/docs/verifiable-document/document-store/).
+| Type  | Name                  | Value                                                         |
+|-------|-----------------------|---------------------------------------------------------------|
+| `TXT` | `openattestation.com` | `openatts net=ethereum netId=3 addr=<DOCUMENT_STORE_ADDRESS>` |
+
+The `<DOCUMENT_STORE_ADDRESS>` in the `Value` field above is the document store smart contract address obtained [in the previous guide](/docs/verifiable-document/document-store/).
 
 An example of a valid `TXT` record is as shown:
 
-```text
-openatts net=ethereum netId=3 addr=0x9db35C07350e9a16C828dAda37fd9c2923c75812
-```
+| Type  | Name                       | Value                                                         |
+|-------|----------------------------|---------------------------------------------------------------|
+| `TXT` | `demo.openattestation.com` | `openatts net=ethereum netId=3 addr=0x9db35C07350e9a16C828dAda37fd9c2923c75812` |
 
 ## Testing the DNS Record
 
 ![Google DNS to Test](/docs/verifiable-document/dns-proof/google-dns.png)
 
-> After the update, you may need to wait up to 24 hrs for the DNS to propagate. You may continue with the other parts of the guide while waiting for DNS to propagate.
+> After the update, you may need to wait up to 24 hours for the DNS to propagate. You may continue with the other parts of the guide while waiting for DNS to propagate.
 
-After adding the `TXT`record, you may check that the record has been inserted correctly by viewing with https://dns.google.com/.
+After adding the `TXT` record, you may want to check that the record has been inserted correctly by viewing with [Google DNS](https://dns.google.com/).
 
-Make sure to select `TXT` in the "RR Type" dropdown.
+Make sure to select `TXT` in the `RR Type` dropdown.
 
 ## Additional Note for Identity Proof in Production
 
-The `TXT` record above is for use for documents issued on the `ropsten` network. To bind the identity in production where your documents are issued in the Ethereum main net, you will have to change `netId` to `1`.
+The `TXT` record above is for use for documents issued on the Ethereum `ropsten` network. To bind the identity in production where your documents are issued in the Ethereum mainnet, you will have to change `netId` to `1`. The `netId` corresponds to the [network ID for the different Ethereum networks](https://chainid.network/).
+
+Throughout our documentation, we generally use only the following networks:
```

Comment:
hum it might be worth to read the whole doc from the start to make sure this is at the correct place :thinking:

Reply:
i edited the phrasing a bit so that it flows. i'm not sure where else to put it though! https://deploy-preview-26--laughing-cori-52b566.netlify.app/docs/verifiable-document/dns-proof#additional-note-for-adding-dns-txt-records